x/revenue/types/params.go

```diff
@@ -4,14 +4,17 @@ import (
 	fmt "fmt"
 
 	"cosmossdk.io/math"
+	"github.com/neutron-org/neutron/v6/app/params"
 	"gopkg.in/yaml.v2"
 )
 
 var (
-	// DefaultBaseCompensation represents the default compensation amount in USD.
-	DefaultBaseCompensation uint64 = 2500
-	// DefaultPaymentScheduleType represents the default payment schedule type.
-	DefaultPaymentScheduleType = &PaymentScheduleType_EmptyPaymentScheduleType{EmptyPaymentScheduleType: &EmptyPaymentScheduleType{}}
+	// DefaultRewardAsset represents the default reward asset.
+	DefaultRewardAsset = params.DefaultDenom
+	// DefaultRewardQuoteAmount represents the default reward amount measured in quote asset.
+	DefaultRewardQuoteAmount uint64 = 2500
```

Comment:
3000

Reply:
would make sense to change the value here https://github.com/neutron-org/neutron/blob/5424008f69175a311211d90fc05137fd9900aac8/app/upgrades/v6.0.0/upgrades.go#L197 instead of the default value to avoid tests fixing. also I see no points in storing the proper mainnet values as the default ones in the module